internal/transform/offer.go

```diff
@@ -0,0 +1,88 @@
+package transform
+
+import (
+	"fmt"
+
+	ingestio "github.com/stellar/go/exp/ingest/io"
+	"github.com/stellar/go/xdr"
+)
+
+//TransformOffer converts an account from the history archive ingestion system into a form suitable for BigQuery
+func TransformOffer(ledgerChange ingestio.Change) (OfferOutput, error) {
+	ledgerEntry := ledgerChange.Post
+	if ledgerChange.LedgerEntryChangeType() == xdr.LedgerEntryChangeTypeLedgerEntryRemoved {
+		ledgerEntry = ledgerChange.Pre
+	}
+
+	offerEntry, offerFound := ledgerEntry.Data.GetOffer()
```

Comment:
Is there a possible nil pointer error when referencing data before calling `GetOffer`?